content/learn/getting-started/node-with-typescript.en.md

```diff
@@ -55,10 +55,14 @@ Now we can compile it to JavaScript using `tsc` command in the terminal. Let's d
 ```bash
 npx tsc example.ts
 ```
+> [npx](https://www.npmjs.com/package/npx) here stands for Node Package Execute. This tool allows us to run TypeScript's compiler without installing it globally.
+
+`tsc` is the TypeScript compiler which will take our TypeScript code and compile it to JavaScript.
 
 This command will result in a new file named `example.js` that we can run using Node.js.
 Now when we know how to compile and run TypeScript code let's see TypeScript bug-preventing capabilities in action!
 
+
```

Comment:
Why this extra line?

Reply:
I thought of explaining tsc in a separate line as I wanted to explain the command and then proceed with the output rather than having them both together. Should I change it?

![image](https://user-images.githubusercontent.com/84540554/199342699-2e90a11b-5247-4270-8933-b4475dd2e4ba.png)

![image](https://user-images.githubusercontent.com/84540554/199342886-7e0261aa-0ac9-4e28-941a-81a7312831eb.png)

Reply:
I think it should be together, no extra line 👀